src/components/vue-form-builder.vue

```diff
@@ -1,389 +1,393 @@
 <template>
-  <div class="dynaform-builder">
-    <div class="palette-container">
-      <div class="card-header">Controls</div>
-      <draggable
-        id="controls"
-        v-model="controls"
-        :options="{sort: false, group: {name: 'controls', pull: 'clone', put: false}}"
-        :clone="cloneControl"
-      >
-        <div class="control" v-for="(element, index) in controls" :key="index">
-          <div class="icon">
-            <img v-if="element['editor-icon']" :src="element['editor-icon']">
-            <i v-if="element['fa-icon']" :class="element['fa-icon']"></i>
-          </div>
-          <div class="label">{{element.label}}</div>
+    <div class="dynaform-builder">
```

Comment:
dynaform? We don't use this. We need to use ONLY bootstrap classes.

Reply:
![image](https://user-images.githubusercontent.com/535300/53917118-ae596e00-4018-11e9-8ba6-015379bb3ec3.png)